src/libostree/ostree-fetcher-util.h

```diff
@@ -33,6 +33,9 @@ G_BEGIN_DECLS
  */
 #define OSTREE_FETCHER_USERAGENT_STRING (PACKAGE_NAME "/" PACKAGE_VERSION)
 
+gboolean
+_ostree_fetcher_validate_mirrorlist (GPtrArray *mirrorlist, GError **error);
```

Comment:
Missing a `git add` for this bit?

Reply:
Ah that's cruft, I realized it wasn't needed since we do (phew) check the returned mirrorlist URIs.